plugins/modules/synciqjob.py

```diff
@@ -51,7 +51,7 @@
 notes:
 - There is delay in the actual state change of the SyncIQ job. The state
   change of jobs in 'scheduled' state is not supported.
-- To start the SyncIQ job use the M(dellemc.powerscale.synciqpolicy) module.
```

Comment:
FQCN - needs to provide for module name otherwise sanity might fail

Reply:
UT's seem to have failed